includes/class-sensei-utils.php

```diff
@@ -2513,6 +2513,39 @@ public static function show_course_lessons( $course_id ) {
 		 */
 		return apply_filters( 'sensei_course_show_lessons', true, $course_id );
 	}
+
+	/**
+	 * Check if a user is the lesson author.
+	 *
+	 * @since 3.0.2
+	 *
+	 * @param int $lesson_id
```

Comment:
Missing a description for the parameters and return value.

Reply:
Done!